src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/GjennbrukDataRevurderingService.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.tilleggsstonader.sak.behandling
+
+import no.nav.familie.prosessering.internal.TaskService
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.libs.unleash.UnleashService
+import no.nav.tilleggsstonader.sak.behandling.barn.BarnService
+import no.nav.tilleggsstonader.sak.behandling.barn.NyttBarnId
+import no.nav.tilleggsstonader.sak.behandling.barn.TidligereBarnId
+import no.nav.tilleggsstonader.sak.behandling.domain.Behandling
+import no.nav.tilleggsstonader.sak.vilkår.stønadsperiode.StønadsperiodeService
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.VilkårService
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeService
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.UUID
+
+@Service
+class GjennbrukDataRevurderingService(
+    val taskService: TaskService,
+    val barnService: BarnService,
+    val vilkårperiodeService: VilkårperiodeService,
+    val stønadsperiodeService: StønadsperiodeService,
+    val vilkårService: VilkårService,
+    val unleashService: UnleashService,
+) {
+
+    @Transactional
+    fun gjenbrukData(behandling: Behandling, forrigeBehandlingId: UUID) {
```

Comment:
Vi burde kanskje kalle forrigeBehandlingId for noe annet, eks "gjenbrukDataFraBehandlingId"

Reply:
Når er det ikke forrigeBehandling da? Så lenge forrige er ferdigstilt brukes vel den?